packages/client/src/generated/ethereum/api.ts

```diff
@@ -276,7 +282,7 @@ export const V1ApiAxiosParamCreator = function (configuration?: Configuration) {
             assertParamExists('estimateGas', 'to', to)
             // verify required parameter 'value' is not null or undefined
             assertParamExists('estimateGas', 'value', value)
-            const localVarPath = `/api/v1/estimate-gas`;
+            const localVarPath = `/api/v1/gas/estimate`;
             // use dummy base URL string because the URL constructor only accepts absolute URLs.
```

Comment:
would it make more sense to do gas estimation in the client?

Reply:
:shrug: either or would work. was already added for chain adapters to use so just carrying it forward